module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -395,10 +395,16 @@ public function init()
                 'Token taken from ' . $cacheType . ' cache: ' . substr($this->token, 0, 30) . '...'
             );
         }
-        if ($this->token == null) {
-            $this->renewTenantToken();
-        } else {
-            $this->checkTenantToken();
+        try {
+            if ($this->token == null) {
+                $this->renewTenantToken();
+            } else {
+                $this->checkTenantToken();
+            }
```

Comment:
Maybe this is over-optimization, but part of me is tempted to reduce these five lines of code down to one:

```suggestion
            ($this->token == null) ? $this->renewTenantToken() : $this->checkTenantToken();
```

Reply:
I disagree with this transformation: to me, the initial one (which I didn't change) is easier to read. I only use the ternary operator to assign a value, to avoid repeating the destination.